### PR TITLE
Add cockroachdb tests

### PR DESCRIPTION
**Pull request by yoshuawuyts**
WIP cockroachdb tests. Ref #900

### CODE REVIEW

File: query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala [hunk 1]
```diff
@@ -123,6 +124,22 @@ object ConnectorConfig {
     }
   }
 
+  lazy val $cockroachdb_Host = {
```
Comment (mavilein) [inline query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala:127]: The variable name should be without `$`. Only when you reference it in an interpolated string like above you use the `$`.

File: query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala [hunk 2]
```diff
@@ -123,6 +124,22 @@ object ConnectorConfig {
...
+    if (EnvVars.isBuildkite) {
+      "test-db-cockroachdb"
+    } else {
+      "127.0.0.1"
+    }
+  }
+
+  lazy val $cockroachdb_Port = {
```
Comment (mavilein) [inline query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala:135]: same as above.

**Review by mavilein — COMMENTED** [review]
It is probably handy if you would test this branch against the corresponding branch of the build CLI. I had to do this recently myself. Here is a commit that shows how to do it by referencing the branch name and commit hash: https://github.com/prisma/prisma-engines/pull/1008/commits/45629525a2026a1026d56336ed17a42b041620bf#diff-224d12340b2e2daae46ebd940fffcb52R16

**Comment by yoshuawuyts** [comment]
Filed https://github.com/prisma/engineer/pull/22 which fixes the CockroachDB side of things. I feel like I'm missing something obvious, as the tests still report they can't connect to the database -- although I've manually verified `engineer` does in fact start the database and the web UI is responsive on localhost.

**Comment by yoshuawuyts** [comment]
@dpetrick CI is failing because it can't download the right version of `engineer`. I thought we were auto-building for every patch; but not sure if that's the case?  Do we have documentation anywhere how to publish new `engineer` versions to AWS?

**Comment by yoshuawuyts** [comment]
CI is failing as expected: it seems the backing store for cockroach behaves different from Postgres.

**Pull request closed without merging**